reddit/reddit-connector/reddit-connector-1.0.0/org.wso2.carbon.connector/src/main/assembly/assemble-connector.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Fixed in https://github.com/wso2-attic/esb-connectors/commit/c61da2ff6de0ced270ed9698ed4fbd8227a92f5b